stdlib/2.7/os/__init__.pyi

```diff
@@ -134,7 +134,16 @@ def rmdir(path: unicode) -> None: ...
 # TODO(MichalPokorny)
 def stat(path: unicode) -> Any: ...
 
-# TODO: stat_float_times, statvfs, tempnam, tmpnam, TMP_MAX, walk
+# TODO: stat_float_times, statvfs, tempnam, tmpnam, TMP_MAX
+def walk(top: AnyStr, topdown: bool = ..., onerror: Any = ...,
+         followlinks: bool = ...) -> Iterator[Tuple[AnyStr, List[AnyStr],
+                                                      List[AnyStr]]]: ...
```

Comment:
Agreed this seems weird, I'll fix this up in the 3/ library too since I just copied this block from there (there isn't a good way to share code here, is there?)